redmine

```diff
@@ -36,11 +36,15 @@ def create_version(args: argparse.Namespace) -> None:
         raise SystemExit(str(ex))
 
 
+def match_formatted_version(version_name, args_name, project_id):
+    return version_name == args_name or version_name == (project_id[0].upper() + project_id[1:] + ' ' + args_name)
```

Comment:
You can at least simplify this to:
```suggestion
    return version_name in (args_name, version_name == (project_id[0].upper() + project_id[1:] + ' ' + args_name))
```

But then you also observe that for every version that's found you construct the string. It would be better to store the potential names in some variable and match against that. See the next comment below: